scripts/dt_spark.sh

```diff
@@ -34,7 +34,7 @@ echo "To start press any key, to stop press Ctrl+C"
 read -n 2
 
 $spark \
-    --class=dt.WordSimFromCounts \
+    --class=de.tudarmstadt.lt.jst.dt.WordSimFromCounts \
```

Comment:
Please modify this script in a such a way that: 

1. it takes an extra parameter that is "conll", "text", or "termcontext". depending on the value. if the first option is selected, then conll job first is launched (main function), for the second option the trigram job is first launched. 

currently, just comment call to "de.tudarmstadt.lt.jst.dt.WordSimFromCounts" as you do not have a class that takes the term context as input.